src/neural/CL/OpenCL.cc

```diff
@@ -530,123 +186,87 @@ void OpenCL_Network::forward(const std::vector<net_t>& input,
       if (niter->is_residual_block) {
         skip_next_in_trans = true;
       }
-      convolve3(layer.channels,
-                layer.outputs,
-                inBuffer,
-                inBuffer,
-                VBuffer,
-                MBuffer,
-                conv_weights,
-                nullptr,
-                bn_weights,
-                skip_in_trans, skip_next_in_trans, true);
+      convolve3(layer.channels, layer.outputs, inBuffer, inBuffer, VBuffer,
+                MBuffer, conv_weights, nullptr, bn_weights, skip_in_trans,
+                skip_next_in_trans, true);
```

Comment:
Undecided about which is better here...

Reply:
again, It's clang-format google.

Reply:
I did not indent/reindent this file by hand.